values.schema.json

```diff
@@ -18,7 +18,143 @@
                       "additionalProperties": false,
                       "properties": {
                         "airflow": {
-                          "type": "object"
+                          "allOf": [
+                            {"$ref": "https://raw.githubusercontent.com/apache/airflow/main/chart/values.schema.json"},
+                            {
+                              "additionalProperties": false,
+                              "properties": {
+                                "affinity": {},
```

Comment:
these were pulled using the following command line script

`ag ".Values.\w+" -o --no-filename --no-numbers | sort | uniq `

Reply:
I think this is our only addition, and it should probably by allowed here as well:

https://github.com/astronomer/airflow-chart/blob/1909e227e1fd8539febc000e763ba63c20c361f6/values.yaml#L334-L335

Reply:
added as well as podMutation as discussed in PM

Reply:
Probably worth documenting these 2 explicitly, or how to find the delta.